<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerWdg</class>
 <widget class="QWidget" name="ColorPickerWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ColorPickerWdg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>误差范围：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>7</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="cb_errAllArea">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>总面积：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_errAllAreaMin">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_errAllAreaMax">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="cb_errSingleArea">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>单面积：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_errSingleAreaMin">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_errSingleAreaMax">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>2</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>属性：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QPushButton" name="bt_selectROI">
             <property name="text">
              <string>选取目标区域</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="bt_selectMinusROI">
             <property name="text">
              <string>选取排除区域</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="cob_currentRoiIndex"/>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cob_currentMinusIndex"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QRadioButton" name="rb_useSelfReg">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>自选区域参考</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_useRefer">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>参考模组：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="rb_useAreaRefer">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>传入面区域参考</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QRadioButton" name="rb_usePointRefer">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string>传入点区域参考</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>区域胀缩：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_AreaZoom">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="minimum">
              <number>-50</number>
             </property>
             <property name="maximum">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="cb_drawAllRegion">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>选取时画出所有区域</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>目标区域数目：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sp_RoiNum">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>排除区域数目：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="sp_MinusNum">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="rb_rec">
                <property name="text">
                 <string>矩形</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_cir">
                <property name="text">
                 <string>圆形</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="cb_notLess">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>不少于总面积</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="cb_showRegionAfter">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>测后显示区域</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="cb_deleteLightArea">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>排除过亮区域</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="cb_fillHole">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>填充孔洞</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>亮阈值：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_lightAreaThreshold">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>亮面积：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sp_lightArea">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QCheckBox" name="cb_height">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>高度:    </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sp_heightMin">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="sp_heightMax">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cb_Width">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>宽度:    </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sp_WidthMin">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="sp_WidthMax">
             <property name="maximum">
              <number>99999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>面积: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sp_areaMin">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="sp_areaMax">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>饱和度:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sp_saturationMin">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="sp_saturationMax">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>色调:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="sp_hueMin">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QSpinBox" name="sp_hueMax">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="cb_open">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>开操作： </string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="sp_open">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>21</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="cb_close">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>闭操作： </string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="sp_close">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="buttonSymbols">
              <enum>QAbstractSpinBox::UpDownArrows</enum>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>21</number>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QPushButton" name="bt_testing">
             <property name="statusTip">
              <string>在第一个roi内进行动态的参数测试</string>
             </property>
             <property name="whatsThis">
              <string>在第一个roi内进行动态的参数测试</string>
             </property>
             <property name="text">
              <string>测试模式</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>groupBox</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>verticalSpacer</zorder>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>6</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
